src/controllers/cgnOperatorSearchController.ts

```diff
@@ -80,6 +82,24 @@ export default class CgnOperatorSearchController {
       );
     });
 
+  /**
+   * Search CGN merchants/discounts that matches with search criteria
+   */
+  public readonly search = (
+    req: express.Request
+  ): Promise<
+    | IResponseErrorValidation
+    | IResponseErrorInternal
+    | IResponseErrorNotFound
```

Comment:
404 is not listed as possible status code

Reply:
fixed in https://github.com/pagopa/io-backend/pull/1161/commits/ac49ea4395eae8b562fcb467639c955239f0996e